test/state.cpp

```diff
@@ -94,11 +94,65 @@ TEST_CASE_FIXTURE(StateTest, "Two Person")
   verify_group_functionality(group);
 }
 
+TEST_CASE_FIXTURE(StateTest, "SFrame Parameter Negotiation")
+{
+  // Set the SFrame parameters for the group
+  auto specified_params = SFrameParameters{ 1, 4 };
+  auto group_extensions = ExtensionList{};
+  group_extensions.add(specified_params);
+
+  // Create the initial state of the group
+  auto first0 = State{ group_id,          suite,           init_privs[0],
+                       identity_privs[0], key_packages[0], group_extensions };
+
+  // Get a KeyPackage from the second member, and verify compatiblity
+  auto compatible_capabilities = SFrameCapabilities{ { 1, 2, 3, 4 } };
+  REQUIRE(compatible_capabilities.compatible(specified_params));
+
+  auto incompatible_capabilities = SFrameCapabilities{ { 2, 3, 4 } };
+  REQUIRE_FALSE(incompatible_capabilities.compatible(specified_params));
```

Comment:
isn't 4 common between the lists here..

Reply:
No, `4` is the value for `epoch_bits`.  I'll change it to be clearer, though.